src/packaging/Config.cmake.in

```diff
@@ -1,8 +1,16 @@
 @PACKAGE_INIT@
 
 include(CMakeFindDependencyMacro)
-find_dependency(ortools)
-find_dependency(minizip)
+find_dependency(ortools REQUIRED)
+find_dependency(minizip-ng)
+if (minizip-ng_FOUND)
+	add_library(MINIZIP::minizip ALIAS MINIZIP::minizip-ng)
+else ()
+	find_dependency(minizip)
+	if (NOT minizip_FOUND)
+		message (FATAL_ERROR "Minizip dependency (minizip or minizip-ng) not found.")
```

Comment:
specify minimal version? according to this [commit](https://github.com/AntaresSimulatorTeam/Antares_Simulator/commit/27de5803089987b907cd136d4277a9d53bd6040c#diff-d0264d71f2ec4efa0918b170f9e350b879ab24b695165a9f41358e74bf2b8fe8), at least v 3.0.7 and back are incompatible with Simulator.
minizip target is confusing it actually refers to [minizip-ng](https://github.com/zlib-ng/minizip-ng?tab=readme-ov-file) and not the ["original"](https://github.com/madler/zlib/tree/master/contrib/minizip) project